File: android/src/main/java/com/skylabs/mixer/RequestParameters.java
package com.skylabs.mixer;

/**
 * local constant mappings for request parameters
 */
public class RequestParameters {
    // BaseMixerRequest
    public static String audioId = "audioId";

    // InitChannelRequest
    public static String filePath = "filePath";
    public static String channelNumber = "channelNumber";
    public static String bassGain = "bassGain";
    public static String bassFrequency = "bassFrequency";
    public static String midGain = "midGain";
    public static String midFrequency = "midFrequency";
    public static String trebleGain = "trebleGain";
    public static String trebleFrequency = "trebleFrequency";
    public static String volume = "volume";
    public static String channelListenerName = "channelListenerName";
    public static String elapsedTimeEventName = "elapsedTimeEventName";

    // AdjustVolumeRequest
//    public static String volume = "volume";
    public static String inputType = "inputType";

    // AdjustEqRequest
    public static String eqType = "eqType";
    public static String gain = "gain";
    public static String frequency = "frequency";
//    public static String inputType = "inputType";

    // ChannelPropertyRequest
//    public static String inputType = "inputType";

    // SetEventRequest
    public static String eventName = "eventName";

    // InitAudioSessionRequest
    public static String inputPortType = "inputPortType";
    public static String ioBufferDuration = "ioBufferDuration";
    public static String audioSessionListenerName = "audioSessionListenerName";

}

File: ios/Plugin/RequestParameters.swift
//
//  RequestParameters.swift
//  Plugin
//
//  Created by Skylabs Technology on 8/13/21.
//  Copyright © 2021 Max Lynch. All rights reserved.
//

import Foundation

/**
 * local constant mappings for request parameters
 */
public class RequestParameters {
    // BaseMixerRequest
    static var audioId: String = "audioId";

    // InitChannelRequest
    static var filePath: String = "filePath";
    static var elapsedTimeEventName: String = "elapsedTimeEventName";
    static var channelNumber: String = "channelNumber";
    static var bassGain: String = "bassGain";
    static var bassFrequency: String = "bassFrequency";
    static var midGain: String = "midGain";
    static var midFrequency: String = "midFrequency";
    static var trebleGain: String = "trebleGain";
    static var trebleFrequency: String = "trebleFrequency";
    static var volume: String = "volume";
    static var channelListenerName: String = "channelListenerName";

    // AdjustVolumeRequest
    // static var volume = "volume";
    static var inputType: String = "inputType";

    // AdjustEqRequest
    static var eqType: String = "eqType";
    static var gain: String = "gain";
    static var frequency: String = "frequency";
    // static var inputType = "inputType";

    // ChannelPropertyRequest
    // static var inputType = "inputType";

    // SetEventRequest
    static var eventName: String = "eventName";

    // InitAudioSessionRequest
    static var inputPortType: String = "inputPortType";
    static var ioBufferDuration: String = "ioBufferDuration";
    static var audioSessionListenerName: String = "audioSessionListenerName";
    
    // FileValidationRequest
    // static var filePath: String = "filePath";

    // StreamRequest
    static var streamUrl: String = "streamUrl";
}
